.changeset/selfish-queens-rest.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': major
+---
+
+Remove Escrow constracts (Escrow, ConditionalEscrow, RefundEscrow and PullPayment).
```

Comment:
I think this needs rephrasing, we're keeping PullPayment and renaming Escrow into PullPaymentEscrow

```suggestion
Remove ConditionalEscrow, RefundEscrow and rename Escrow into PullPaymentEscrow.
```

Reply:
We ended up re-removing PullPayment and Escrow. Do you want to update your suggestion ?

Reply:
```suggestion
Remove PullPayment and Escrow contracts (Escrow, ConditionalEscrow, RefundEscrow).
```

Reply:
I'd keep @frangio's suggestion

Reply:
It was applied.